config.env

```diff
@@ -0,0 +1 @@
+SECRET = jenssecretforstudentprojectthisisit
```

Comment:
This shouldn't be on github :-*

Reply:
:scream:
deleted file. Will update .gitignore and secret on next PR